loadgen/agent-trade-amm.js

```diff
@@ -22,6 +22,25 @@ export default async function startAgent([key, home]) {
     bldPurse: E(wallet).getPurse(pursePetnames.BLD),
   });
   // const bldBrand = await E(bldPurse).getAllegedBrand();
+
+  {
```

Comment:
I think this clause should live in `doSetupTransfer`, to maintain the pattern of "define a bunch of functions" followed by "invoke some of them", and so all the setup stuff happens in one obvious place.

Reply:
It cannot, it needs to happen before the `E(zoe).getPublicFacet(autoswap)`. That's where it was before https://github.com/Agoric/testnet-load-generator/pull/17/commits/eaafa4b9d709f09ea2a14c8798425916f7a863b4